container/go/cmd/pusher/pusher.go

```diff
@@ -33,7 +34,7 @@ import (
 var (
 	dst             = flag.String("dst", "", "The destination location including repo and digest/tag of the docker image to push. Supports fully-qualified tag or digest references.")
 	src             = flag.String("src", "", "Path to the directory which has the image index that will be pushed.")
```

Comment:
Update help text here. Suggest `Path to the manifest.json when -format is legacy, path to the index.json when -format is oci or path to the image .tar file when -format is docker`.